pibooth/camera/base.py

```diff
@@ -17,12 +17,14 @@ def __init__(self, camera_proxy):
         self._captures = []
 
         self.resolution = None
+        self.preview_resolution = None
         self.delete_internal_memory = False
         self.preview_rotation, self.capture_rotation = (0, 0)
         self.preview_iso, self.capture_iso = (100, 100)
         self.preview_flip, self.capture_flip = (False, False)
+        self.custom_fourcc = "Default"
```

Comment:
Same kind of comment: the `fourcc` part shall not be in the `base` camera

Reply:
Yes you're right, i will try to move it to OpenCV class

Reply:
If I want to mode some opencv specific configuration values, i think i will need to refactor the configuration of the cameras. Currentyl, some "general" parameters like iso, resolution (delete_internal_memory <== Specific paramter) are passed directly) to the initialize method. But all the cameras does not share the same parameters. IMHO it will be better  to passe the cfg to each derived class and each class will retrieve the custom parameters according to its need . How do you think about this solution ?

Reply:
Indeed, you are right we already have exception and it needs a bigger change to work differently. I think the best is to avoid making a bigger change in this PR and keep it like it, maybe just be more explicit on the parameters name like `opencv_use_v4l2` and `opencv_custom_fourcc` or at least be explicit in the config.py file that these parameters are used only for opencv cameras